### ISSUE TITLE
Support for Symfony 5?

### ISSUE THREAD

**Issue by danaki** [skrajewski/resource-naming-strategy#4]
Getting this:
```
!!  Fatal error: Uncaught ArgumentCountError: Too few arguments to function Symfony\Component\Config\Definition\Builder\TreeBuilder::__construct(), 0 passed in C:\work\lionel-api\vendor\szykra\resource-naming-strategy\src\ResourceNamingBundle\DependencyInjection\Configuration.php on line 20 and at least 1 expected in C:\work\lionel-api\vendor\symfony\config\Definition\Builder\TreeBuilder.php:26
```

**Comment by skrajewski** [comment]
Hi @danaki,

I haven't tested this package with Symfony 5. Apparently something changed in the last version. I'll look into this when I have some free time, but if you find the solution earlier, feel free to send PR. 

Thanks!

**Comment by danaki** [comment]
I think the only change needed to fix is the required `name` parameter of `TreeBuilder`. I guess it affects the configuration such as yaml configs but I can't find any config of your bundle in my project. So I wonder why this is needed anyway and thus I doubt I can appropriately fix it just because I don't understand it.

**Comment by danaki** [comment]
Please merge this https://github.com/skrajewski/resource-naming-strategy/pull/6/commits/f641e941fc153601e39da463131c8251de6d3186 as well. I should have created another branch for this feature.

**Comment by skrajewski** [comment]
Fixed by #6 in version 0.1.4
